mobile/lib/components/newest_recording/newest_recording_widget.dart

```diff
@@ -0,0 +1,45 @@
+import 'package:flutter/cupertino.dart';
+import 'package:radioaktywne/components/utility/swipeable_card.dart';
+import 'package:radioaktywne/extensions/build_context.dart';
+
+/// Widget representing three most recent recordings.
```

Comment:
add a TODO comment explaining this is a dummy widget - it will make it easier to make the app test-ready